src/components/ControlledInputFields/ControlledSelectInputField/ControlledSelectInputField.js

```diff
@@ -19,14 +19,13 @@ const useStyles = makeStyles((theme) => ({
 }))
 
 const ControlledSelectInputField = ({
+  isRequired = false,
   control,
   name,
   label,
-  placeholder,
-  options,
-  customValidationFunctions,
   error,
-  required,
+  customValidationFunctions = {},
+  SelectProps = { placeholder: "" },
```

Comment:
You don't need to provide a default value of an empty string for the `placeholder` prop